app/src/androidTest/java/net/skyscanner/backpack/button/BpkButtonLinkTest.kt

```diff
@@ -0,0 +1,104 @@
+package net.skyscanner.backpack.button
+
+import android.widget.FrameLayout
+import androidx.core.content.ContextCompat
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.skyscanner.backpack.BpkSnapshotTest
+import net.skyscanner.backpack.createThemedContext
+import net.skyscanner.backpack.demo.R
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+
+/**
+ * FIXME
+ * 31/10/2018
+ * see https://github.com/Skyscanner/backpack-android/pull/103
```

Comment:
No need on that, it's closed

Reply:
It is still relevant, the PR linked here is where it was introduced but the bug is still present. We cannot test button with icons because it doesn't work with relative position.